env/lib/python3.12/site-packages/pip/_internal/models/index.py

```
@@ -0,0 +1,28 @@
+import urllib.parse
+
+
+class PackageIndex:
+    """Represents a Package Index and provides easier access to endpoints"""
+
+    __slots__ = ["url", "netloc", "simple_url", "pypi_url", "file_storage_domain"]
```

Ruff (0.13.3)

`PackageIndex.__slots__` is not sorted

Apply a natural sort to `PackageIndex.__slots__`

(RUF023)